receiver/gitproviderreceiver/documentation.md

```diff
@@ -34,6 +34,79 @@ Number of repositories in an organization
 | ---- | ----------- | ---------- |
 | 1 | Gauge | Int |
 
+### git.repository.pull_request.approved.time
+
+The amount of time it took a pull request to go from open to approved
+
+| Unit | Metric Type | Value Type |
+| ---- | ----------- | ---------- |
+| 1 | Gauge | Int |
```

Comment:
Looks like all the metric unit fixes have been lost somewhere in the rebases/merges? See [this comment](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29642#pullrequestreview-1794151974) on this PR and the changes below that comment.

```suggestion
| s | Gauge | Int |
```